_persons/thanhson.md

```diff
@@ -0,0 +1,28 @@
+---
+# use person if you want to have a personal website created
+layout: person
+first_name: Thanh Son
+last_name: Nguyen
+key: nguyen
+permalink: /team/nguyen/
+# faculty | phdstudent | phd_rotation | postdoc | msstudent | collaborator
+role: phdstudent
+email: thanhson@cs.utah.edu
+image: /assets/images/people/thanhson.jpg
```

Comment:
The size of your headshot image should be 400x400 pixels. Sorry, I did not notice that sooner.

Reply:
I updated the image